### DISCUSSION TITLE
Move away from colors.js

### DISCUSSION THREAD

**Discussion by mufty** [ReactiveX/rxjs#6745] [Ideas / Feature request]
Please swap the colors.js library for something else it's not safe to use with the dev behind it.

See: https://twitter.com/BleepinComputer/status/1480182019854327808?s=20

**Comment by ryust-ax** [comment]
I tracked down recently updated forks and found two promising future-maintainer candidates:
- https://github.com/euberdeveloper/colors.js
- https://github.com/mobilehero-archive/titanium-colors

Incidentally, good tool for this was https://techgaun.github.io/active-forks/index.htm

**Comment by benlesh** [comment]
Interesting. Is this causing us issues anywhere? I haven't seen any problems yet.

**Reply by kwonoj** [reply]
we have pkglock so not affected. actually I don't know why we have it as dep even.

**Reply by benlesh** [reply]
it's a dependency of `validate-commit-msg`. My guess is it was updated at some point to solve some dep conflict or maybe something with dependabot, and that added it as a dev dependency for us.

**Reply by benlesh** [reply]
![image](https://user-images.githubusercontent.com/1540597/149217416-591625d8-e5e0-4be6-ac56-3cc16e1ac4c4.png)

**Reply by kwonoj** [reply]
@benlesh I'm fine with transitive dep. What I meant is why we have it as direct dep: https://github.com/ReactiveX/rxjs/blob/master/package.json#L161-L162

**Reply by kwonoj** [reply]
..and we have `color`, `colors`? 🤔
